build-coordinator/src/main/java/org/jboss/pnc/core/builder/BuildTask.java

```diff
@@ -188,7 +204,14 @@ public String getBuildContentId() {
     }
 
     void addWaiting(BuildTask buildTask) {
```

Comment:
I agree, I left it there at first for compatibility, but it's better to just remove it.